docs/ConnectPOB/manage-nodes.md

```diff
@@ -83,3 +140,116 @@ When clicking a node item this extended information that is attached to the node
+|     A     | The given name of the node.                                                                                                                                                                                                                                                                                                                                                                                                                |
+|     B     | Signal strength to parent node in desibel. This value will alway be minus. Green value -0 to -69 the node has GOOD connection to parent node. Yellow values -70 to - 79 the node has OK connection to parent node, consider adding another node between the two nodes. Red values -80 or worse, the node has BAD connection to parent node and another node should be placed between the two to ensure that all data will be transimitted. |
+|     C     | Nodes mac address                                                                                                                                                                                                                                                                                                                                                                                                                          |
+|     D     | Time since node last reported to parent node. If time since is more than 20 minutes the time will have a orange colour.                                                                                                                                                                                                                                                                                                                    |
+
+#### MDT - Node data 1
+
+This is the chip in the node that handles bluetooth communication.
+![Mesh Debug Tool node data section 1](node-admin-images/mesh-debug-images/mesh-debug-tool-part-2.png)
+
+| List item | Explanation                                                                                                                                                                                                        |
+| :-------: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|     A     | Section Heading                                                                                                                                                                                                    |
```

Comment:
Does not need a screenshot?
Better to list the name of each item in list item column instead of A, B, C?
Then the screenshot is not needed

Reply:
So something like this:?

```MD
|    List item    | Explanation                                                                                          |
| :-------------: | :--------------------------------------------------------------------------------------------------- |
|    Udpseqno     | UDP (User Datagram Protocol) sequence number                                                         |
|     Uptime      | Uptime of node in seconds                                                                            |
|   Nodeversion   | What version is the firmware in the node running. This should be the same as Node data 1.            |
|   Bleversion    | What BLE (Bluetooth Low Energy) version is the node running. This should be the same as Node data 1. |
|  Resetattempts  | How many attempts has there been to reset the node??!?!                                              |
|  Summarycount   | Summary cout.......                                                                                  |
| Secsincesummary | Sec since summary.....                                                                               |
| Bsbuildversion  | BS build-version ......                                                                              |
| Otabuildversion | OTA build version.......                                                                             |
| Rplbuildversion | RPL build-version.......                                                                             |
```
Renders to:
![image](https://github.com/ScanReach/ScanReach.Documentation/assets/5204006/da55035b-0e21-4dad-a837-c1d6b6996aec)